reference/configuration/framework.rst

```diff
@@ -196,9 +197,35 @@ see :doc:`/cookbook/request/load_balancer_reverse_proxy`.
 form
 ~~~~
 
+.. _form-enabled:
+
+enabled
+.......
+
+**type**: ``boolean`` **default**: ``false``
+
+Whether or not to enable support for the Form component.
+
 csrf_protection
 ~~~~~~~~~~~~~~~
 
+.. _csrf-protection-enabled:
+
+enabled
+.......
+
+**type**: ``boolean`` **default**: ``true`` if form support is enabled, ``false``
+otherwise
+
+This option can be used to disable CSRF protection of forms.
```

Comment:
technically, if you never use the CSRF protection in your app (an API-only website for instance), it makes sense to disable it, because it is required to disable it to be able to disable sessions (which is often wanted for APIs)